systems/registry/node/pkg/db/node_repo_test.go

```diff
@@ -163,71 +167,6 @@ func TestNodeRepo_Get(t *testing.T) {
 	})
 }
 
-func TestNodeRepo_GetForOrg(t *testing.T) {
```

Comment:
Is there any reasons why we are removing the test for NodeRepo_GetForOrg ?

Reply:
Yeah we had to clean this as part of registry tear down. Have to figure out better way to do the verfication.

Reply:
Agree!